scripts/windows/specs/PY34-win32.txt

```diff
@@ -0,0 +1,24 @@
+# List packages that are/would be required for installation
+
+--find-links=https://orange.biolab.si/download/files/wheelhouse
+--only-binary numpy,scipy,Bottleneck,scikit-learn,PyQt5
+
+AnyQt==0.0.8
+numpy==1.12.1
+scipy==0.16.1
```

Comment:
If there existed freely available/redistributable scipy binary packages we would not be stuck in this mess in the first place [*].

Also scipy v0.16.1 is not affected by that only 0.17, and 0.18 are.

[*] It actually might be completely fine to use MKL now. Intel changed licence for MKL redistributables to [Intel Simplified Software License](https://software.intel.com/en-us/license/intel-simplified-software-license) which is a lot more friendly.